### PR TITLE
Change 'Username' hint to 'E-Mail' in fragment_signup.xml

### PR DESCRIPTION
**Pull request by namanpreetsingh**
The SignUp screen is now more clear to a new user in terms of what details to enter.

### CODE REVIEW

File: app/src/main/res/layout/fragment_signup.xml [hunk 1]
```diff
@@ -46,7 +46,7 @@
                 android:id="@+id/usernameSignUp"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:hint="@string/user_name"
+                android:hint="@string/e_mail"
```
Comment (dreadpool2) [inline app/src/main/res/layout/fragment_signup.xml]: email is fine :+1:

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -27,6 +27,7 @@
 
     <!--profile details-->
     <string name="user_name">Username</string>
+    <string name="e_mail">E-Mail</string>
```
Comment (dreadpool2) [inline app/src/main/res/values/strings.xml]: Email

Comment (nikit19) [inline app/src/main/res/values/strings.xml]: just `Email` is fine there is already a string for that. You can use it

Comment (namanpreetsingh) [reply app/src/main/res/values/strings.xml]: Corrected.

Comment (namanpreetsingh) [reply app/src/main/res/values/strings.xml]: Corrected.

**Pull request merged**